server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/Mode.java

```diff
@@ -32,7 +32,8 @@
     ASYNC(false),
     ;
     private final boolean sync;
-    private Mode(boolean sync) {
+
+    Mode(boolean sync) {
```

Comment:
An enum constructor is private by definition, so I removed the redundant declaration